# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xfsprogs/autoconf.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


--- xfsprogs-2.6.0/configure.in.orig	2004-01-05 23:02:03.000000000 +0100
+++ xfsprogs-2.6.0/configure.in	2004-01-05 23:02:14.000000000 +0100
@@ -1,6 +1,8 @@
 AC_INIT(include/libxfs.h)
 AC_CONFIG_HEADER(include/platform_defs.h)
 
+AC_PROG_CC
+
 AC_ARG_ENABLE(shared,
 [ --enable-shared=[yes/no] Enable use of shared libraries [default=yes]],,
 	enable_shared=yes)
